docs/issue_registry_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Issue Registry
```

Comment:
Wonder if we should have called it Repairs Issue Registry?

Reply:
We can call it that in the documentation even if the file is `helpers/issue_registry.py`?

Reply:
Yeah, I guess.